### ISSUE TITLE
[Scaffold] Declarative customization of column type

### ISSUE THREAD

**Issue by MaceWindu** [linq2db/linq2db#3603]
all good now, thx!

but please add this feature to the JSON, much easier to work with.
Even with the old version was much easier to change the types of columns.

_Originally posted by @integral-llc in https://github.com/linq2db/linq2db/discussions/3600#discussioncomment-2873826_

**Comment by MaceWindu** [comment]
Currently I don't think it is a good idea to have such low-level customizations in form of configuration. Feedback is welcome

**Comment by Metadorius** [comment]
> Currently I don't think it is a good idea to have such low-level customizations in form of configuration. Feedback is welcome

I disagree. Sometimes, say, enum values are stored in DB as plain integers for various reasons, and the CLI tool has no way of knowing which column should be an enum or an integer.

**Comment by Metadorius** [comment]
Also close example would be the usage of sequences (in Firebird language: generators) for certain columns. Currently there's no way to specify which key should use which column. I've tried to look into interceptors but I am not sure if that can be done via interceptors at all.
